pubsub/google/cloud/pubsub/topic.py

```diff
@@ -435,12 +462,27 @@ def publish(self, message, **attrs):
 
         :type attrs: dict (string -> string)
         :param attrs: key-value pairs to send as message attributes
+
+        :rtype: None
```

Comment:
Very annoying of Pylint to do that. If you change the `return self.commit` to two lines (per my feedback), does Pylint still complain?